### ISSUE TITLE
The built-in viewer shrinks to one side of the screen on my new laptop.

### ISSUE THREAD

**Issue by BlueBirdHouse** [texstudio-org/texstudio#368]
### Environment
- TeXstudio: 2.12.10 (git 2.12.10)
- Qt: 使用 Qt 5.8.0，由 Qt 5.8.0 R 构建
- OS: Windows 10

The built-in viewer shrinks to one side of the screen on my new laptop.

The TeX-studio works fine on my old computer. However, there are some display problems with my new laptop. This new laptop has an Intel Core i5-5200U CPU, so its resolution ratio (1920*1080) is higher than my old one. Everything seems smaller in the screen at the beginning. I use Win10’s zoom settings to display 125% of the default layout. But, the TeX-studio is unable to adopt this setting. The buttons are still too small. The worse thing is that the built-in viewer shrinks to one side of the screen. When I push the full-screen button on the viewer’s window, it changes into a small block and stays at the middle of the screen. To make it works in a real full-screen model, I have to push the button again. 

![1](https://user-images.githubusercontent.com/6317809/47649105-23262280-dbb7-11e8-8e3c-d79d59760ea7.PNG)
![2](https://user-images.githubusercontent.com/6317809/47649106-23beb900-dbb7-11e8-99c9-5c5732ef3f46.PNG)
![3](https://user-images.githubusercontent.com/6317809/47649108-23beb900-dbb7-11e8-9908-267c69826c77.PNG)

**Comment by timhoffm** [comment]
Please try and not mix different topics in a single issue. That makes it hard to follow the discussion.

**GUI Scaling**

Qt cannot handle fractional magnifications very well. You can configure the size of all relevant parts of the GUI:

- `Options -> General -> Appearance -> Font Size`
- `Options -> GUI Scaling`
- `Options -> Editor -> Font Size`
- `Options -> Internal PDF Viewer -> Screen Resolution`

**Viewer**

Sorry, but I cannot completely follow you.

- In your above example is the viewer initially embedded in the main window or in a separate window?
- Technically, we do not have a full-screen button. There is a menu entry `View -> Full Screen` when using the viewer in a separate window. Do you mean that?
- Can you please post a screenshot of your whole screen before and after the operation?

**Comment by sunderme** [comment]
no feedback

**Comment by BlueBirdHouse** [comment]
These are the screenshot that you want, and I am in full-screen mode. 
Unfortunately, this issue is closed. I will open a new one to draw your attention. 

![1](https://user-images.githubusercontent.com/6317809/48296619-00a7d980-e4d4-11e8-9da6-eb85063b3fe5.png)
![2](https://user-images.githubusercontent.com/6317809/48296623-0a314180-e4d4-11e8-8157-4a170c326761.png)
![3](https://user-images.githubusercontent.com/6317809/48296625-0ef5f580-e4d4-11e8-9cd3-3bd2baff928e.png)
![4](https://user-images.githubusercontent.com/6317809/48296630-161d0380-e4d4-11e8-8ac6-69a28b97c6e5.png)
